Reorganise information in lazer-related articles (#9590)
ppy/osu-wiki: wiki/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)/en.md

Author: they look like this on the website and it's intentional

![image](https://github.com/ppy/osu-wiki/assets/36758269/f999a8dc-d1bc-4c5b-bbf8-13ff01ebfc73)

Reviewer: yep I understood that :heart

> # Gameplay differences in osu!(lazer) 

Author: as an article name that sounds not very descriptive

| Intentionally changed | Yes |
| Needs further consideration | Yes |

### Differences in scoring

Scoring in lazer will use a system similar to ScoreV2, with all ScoreV1 scores converted into the new system.

There are two interchangeable display modes for scores: *standardised* and *classic*. Standardised scoring limits score to a maximum of 1,000,000 points + bonus and score multipliers (similarly to ScoreV2), while classic scoring is the same as standardised, but scaled quadratically with the amount of hit objects in a beatmap (similar to ScoreV1). These can be selected from the settings, with all places were score is displayed in-game changing accordingly.

> There are also some differences in how much score each hit object and each judgement is responsible for relative to each other.

Author: ```suggestion
There are also some differences in how much hit objects and judgements contribute to a score relative to each other.
```

> There are also some differences in how much score each hit object and each judgement is responsible for relative to each other.

Reviewer 2: I'm not actually sure what this refers to specifically. Maybe worth elaborating on if it's to remain?

> There are also some differences in how much score each hit object and each judgement is responsible for relative to each other.

Author: this is something on my list to investigate shortly (will report about it against `ppy/osu`, as i mentioned in the meeting), so i'll elaborate in another pass
